lib/main.js

```diff
@@ -30,7 +30,11 @@ module.exports = function (argv) {
     let command = _.first(argv['_'])
     argv['_'] = argv['_'].slice(1)
     if (commands[command]) {
-      yield commands[command](argv)
+      if (command === "clone") {
+        console.log(`"clone" is invalid command for Gather. Use "sync"`)
```

Comment:
We could also delete the `clone.js` file completely

Reply:
YOu already did...